oragono.yaml

```diff
@@ -439,6 +439,10 @@ limits:
     # maximum length of channel lists (beI modes)
     chan-list-modes: 60
 
+    # maximum number of CHATHISTORY messages that can be
+    # requested at once (0 disables support for CHATHISTORY)
+    chathistory-maxmessages: 100
```

Comment:
It feels like we should put this down in the `history` section instead. The main reason for putting things here rather than more specific sections is that most of these entries don't fit nicely anywhere else, whereas since this is the length of the history grabs it makes sense for it to be down there.